### PR TITLE
refactor clusterInfo and clusterClaim

### PR DESCRIPTION
**Pull request by zhiweiyin318**

### CODE REVIEW

File: pkg/klusterlet/clusterinfo/clusterinfo_controller.go [hunk 1]
```diff
@@ -63,9 +62,13 @@ func (r *ClusterInfoReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	// Update cluster info status here.
 	newStatus := clusterv1beta1.ClusterInfoStatus{}
 	var errs []error
+
 	// Config console url
-	_, _, clusterURL := r.getMasterAddresses()
-	newStatus.ConsoleURL = clusterURL
+	newStatus.ConsoleURL, err = r.GetClusterClaim(clusterclaim.ClaimOCMConsoleURL)
```
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply pkg/klusterlet/clusterinfo/clusterinfo_controller.go]: done

File: pkg/klusterlet/clusterinfo/clusterinfo_controller.go [hunk 2]
```diff
@@ -63,9 +62,13 @@ func (r *ClusterInfoReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
...
+	if err != nil {
```
[the remark this answers is not in the document]
Comment (zhiweiyin318) [reply pkg/klusterlet/clusterinfo/clusterinfo_controller.go]: if claim is not found, the corresponding section in clusterinfo will be empty string.

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**